src/components/LinePath/index.tsx

```diff
@@ -9,37 +9,21 @@ import colors from "../../style/colors";
 const getX = (d: DateValueType): Date => new Date(d.date);
 const getY = (d: DateValueType): number => d.value;
 
-const startDate = new Date();
-const defaultArr = [
-  {
-    date: startDate.toISOString(),
-    value: 0,
-  },
-  {
-    date: new Date(startDate.getDate() + 1).toISOString(),
-    value: 0,
-  },
-];
-
-const normalizeData = (data: DateValueType[]): DateValueType[] =>
-  data.length <= 1 ? defaultArr : data;
```

Comment:
Ahh, that's where the issue was, right?

Reply:
Right! It was initially added for the AreaChart so that there is always at least a line when no or only one record exists. The Area Chart has been ported to its own component anyway so this is unnecessary now.